src/applications/hca/containers/IntroductionPage.jsx

```diff
@@ -9,8 +9,8 @@ import SaveInProgressIntro, { introActions, introSelector } from '../../common/s
 import DowntimeNotification, { services, serviceStatus } from '../../../platform/monitoring/DowntimeNotification';
 import AlertBox from '@department-of-veterans-affairs/formation/AlertBox';
 
-function renderDowntime(status, downtimeWindow, downtimeMap, children) {
-  if (status === serviceStatus.down) {
+function renderDowntime(downtime, children) {
+  if (downtime.status === serviceStatus.down) {
     const message = (
```

Comment:
You can pass a `render` prop to `DowntimeNotification` for more controlled responses to downtime/downtime approaching. The downtime parameter contains the status (either down or downtimeApproaching), externalService, startTime (moment object), and endTime (optional moment object).